src/transforms/onehot.jl

```diff
@@ -3,11 +3,17 @@
 # ------------------------------------------------------------------
 
 """
-    OneHot(col)
-    
+    OneHot(col, categorical)
+
 Transforms categorical column `col` into one-hot columns of levels
 returned by the `levels` function of CategoricalArrays.jl.
-
+The `categorical` parameter is a bool type parameter, with `true` as
+default value.
+When `categorical` parameter is:
+  (i)  true:  the one-hot columns are of categorical type with values 
+            of either `true` or `false`
+  (ii) false: the one-hot columns are of bit type with values 
+            of either `1` or `0`
```

Comment:
Be more succinct. People are often in a hurry to learn the basic usage. The less text the better. Just say something like "if categorical is set to `true` the new columns will be categorical arrays".

Reply:
Sure 👍Will try to be more concise.